plugins/services/src/js/reducers/serviceForm/PortMappings.js

```diff
@@ -0,0 +1,150 @@
+import {findNestedPropertyInObject} from '../../../../../../src/js/utils/Util';
+import {
+  ADD_ITEM,
+  SET
+} from '../../../../../../src/js/constants/TransactionTypes';
+import Transaction from '../../../../../../src/js/structs/Transaction';
+
+module.exports = {
+  JSONReducer: null,
+
+  /**
+   * This parser must take precedence over PortDefinition parser, as it assumes
```

Comment:
Parses should not have any dependencies on other parsers. Let's refactor this in the next iteration.